### PR TITLE
updated to be in line with the node webjars

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

**Comment by BrandonArp** [comment]
@jroper This PR just includes the node package for typescript.  I don't think it makes sense to build a package for browser use since this is just a compiler.

**Comment by jamesward** [comment]
I think some people are using the existing `typescript` WebJar so we should add that as a module.

**Comment by BrandonArp** [comment]
Updated this with the web folder as well.

File: pom.xml [hunk 1]
```diff
@@ -38,14 +38,19 @@
...
-        <destDir>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${upstream.version}</destDir>
+        <upstreamVersion>1.4.1</upstreamVersion>
+        <sourceUrl>http://registry.npmjs.org/typescript/-/</sourceUrl>
+        <destDir>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${upstreamVersion}</destDir>
+        <extractDir>${project.build.directory}/typescript.js-${upstreamVersion}</extractDir>
     </properties>
-    
+
+    <modules>
+        <module>node</module>
```
Comment (jamesward) [inline pom.xml]: There needs to be a module for `web` here.

**Comment by BrandonArp** [comment]
Updated again.  I also bumped the version to import to 1.5-beta and only included the .js and .d.ts files in the web module.

**Comment by jamesward** [comment]
Thanks.  One more... `version` in all the poms needs to match the upstream version with `-SNAPSHOT` appended.  So: `<version>1.5.0-beta-SNAPSHOT</version>`

**Comment by BrandonArp** [comment]
Hopefully I've got it this time.

**Comment by jamesward** [comment]
Thanks! This will be released shortly.

**Pull request merged**